packages/react/src/FilteredActionList/FilteredActionList.stories.tsx

```diff
@@ -57,7 +58,7 @@ export function Default(): JSX.Element {
   const filteredItems = items.filter(item => item.text.toLowerCase().startsWith(filter.toLowerCase()))
 
   return (
-    <>
+    <FeatureFlags flags={{primer_react_select_panel_with_modern_action_list: true}}>
```

Comment:
Flagging in this feature cleans up [2 axe violations](https://primer.style/react/storybook/?path=/story/components-filteredactionlist--default):
- ARIA input fields must have an accessible name
- Certain ARIA roles must contain particular children

Reply:
Since the default storybook reflects what's currently GA-ed (with the option to toggle on the feature flag in the storybook toolbar), I don't think we should add this feature flag here.

Maybe we can exclude this violation for now with a note that the fix is behind a feature flag?